### DISCUSSION TITLE
Noseweelstearing works only to one side with the Thrustmaster Airbus stick

### DISCUSSION THREAD

**Discussion by Hor1463** [flybywiresim/aircraft#6424] [Help]
Hi, I doen´t now what happend, but I make all things that are shown in the video from real world sim pilot, but my stick works for the noseweel only to one side. What I have to do?
Roland

**Comment by tracernz** [comment]
Maybe you bound mixture 4 0/100% axis rather than mixture 4 -100/+100% axis?
